index.html

```diff
@@ -457,20 +465,20 @@ <h4>DataIntegrityProof</h4>
           </p>
           <p>
 The `cryptosuite` property of the proof MUST be
-`experimental-quantum-safe-dilithium-2024`.
+`experimental-ml-dsa-2024`.
           </p>
           <p>
-The value of the `proofValue` property of the proof MUST be an Dilithium-2
+The value of the `proofValue` property of the proof MUST be an ML-DSA-44
 signature produced according to using the algorithms specified in section
 [[[#algorithms]]], encoded according
-to ??? encoded using the base-64-url-nopad header and
+to <span class="issue">What is the encoding?</span>??? encoded using the base-64-url-nopad header and
```

Comment:
Encoding is whatever is in FIPS-204.

Reply:
> Encoding is whatever is in FIPS-204.

I checked [FIPS-204](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.204.ipd.pdf) at page 24, and it seems that no encoding is defined. We're using [liboqs](https://github.com/open-quantum-safe/liboqs/tree/main/src/sig/ml_dsa/pqcrystals-dilithium-standard_ml-dsa-44-ipd_ref) which takes binaries as input/output. 

We're currently using base58 for keys and sigs... should we stick to that?

Reply:
My understanding is we would use the pkEncode function to repsent the keys as bytes following FIPS-204 then encode those bytes in some manner.

I think we should use multibase as thats what the other data integrity specs use. Whether base58-btc or base64 I don't know, I think I prefer base58